processing/src/main/java/io/druid/query/search/SearchQueryRunner.java

```diff
@@ -161,7 +161,7 @@ public SearchQueryRunner(Segment segment)
 
             Map<String, DimensionSelector> dimSelectors = Maps.newHashMap();
             for (String dim : dimsToSearch) {
-              dimSelectors.put(dim, cursor.makeDimensionSelector(dim));
+              dimSelectors.put(dim, cursor.makeDimensionSelector(dim, null));
```

Comment:
Sounds good. I'm ok preventing that from being scope creep for this PR, but would you mind adding a TODO note here indicating that desire?